cpp/oneapi/dal/backend/primitives/BUILD

```diff
@@ -44,6 +44,7 @@ dal_collect_modules(
         "search",
         "selection",
         "sort",
+        "sparse_blas",
```

Comment:
Please, add the same line in tests to enable your tests in CI scope.
It should be line#95 in this file(cant comment the line)